runtime/service.go

```diff
@@ -51,7 +50,8 @@ import (
 )
 
 const (
-	defaultVsockPort = 10789
+	defaultVsockPort     = 10789
+	supportedMountFSType = "ext4"
```

Comment:
We should probably allow the user to specify an alternative filesystem (e.g. xfs). But we don't need that immediately.

Reply:
Added ticket for this: https://github.com/firecracker-microvm/firecracker-containerd/issues/44